terraform/udp/healthcare-oie/healthcare-oie_tf_12_9.tf

```diff
@@ -0,0 +1,151 @@
+variable "org_name" {}
+variable "api_token" {}
+variable "base_url" {}
+variable "demo_app_name" { default = "healthcare" }
```

Comment:
Shouldn't this be healcare-oie or healthcareoie?

Reply:
yup. you are correct!